packages/flagging/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@datadog/browser-flagging",
+  "version": "6.7.0",
```

Comment:
This is because at Datadog the convention is for all versions that play together to be kept in sync?

Reply:
It's not a Datadog convention, more like our team convention 😄 

The version will be automatically updated on every release.